variables.tf

```
@@ -0,0 +1,178 @@
+variable "create_iam_roles" {
+  type        = bool
+  description = "Flag to indicate whether to create IAM roles or not"
+  default     = true
+}
+
+variable "iam_roles_name_prefix" {
+  type        = string
+  description = "Name prefix for your EC2 role name and APP role name"
+  default     = ""
+}
+
+variable "ec2_role_name" {
+  type        = string
+  description = "EC2 role name"
+  default     = "aviatrix-role-ec2"
+}
+
+variable "app_role_name" {
+  type        = string
+  description = "APP role name"
+  default     = "aviatrix-role-app"
```

Not setting this variable to allow downstream logic to handle this differently (see  modules/aviatrix-controller-initialize/variables.tf line 71/72 suggestions)

```suggestion
  default     = ""
```